{
    "type": "bundle",
    "id": "bundle--13714133-a603-4b65-a7c5-621c8e40e21d",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--f42feea6-c88c-472f-ae0d-d18dc9b0586a",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-01-24T01:09:40.669024Z",
            "modified": "2025-01-24T01:09:40.669024Z",
            "name": "CVE-2025-22153",
            "description": "RestrictedPython is a tool that helps to define a subset of the Python language which allows to provide a program input into a trusted environment. Via a type confusion bug in versions of the CPython interpreter starting in 3.11 and prior to 3.13.2 when using `try/except*`, RestrictedPython starting in version 6.0 and prior to version 8.0 could be bypassed. The issue is patched in version 8.0 of RestrictedPython by removing support for `try/except*` clauses. No known workarounds are available.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-22153"
                }
            ]
        }
    ]
}